pkg/api/api.go

```diff
@@ -275,11 +273,11 @@ func canaryTestFailures(result map[string]v12.SortedAggregateTestsResult) []v1.C
 }
 
 // job runs with failure groups
-func failureGroupList(report v12.TestReport) []v1.FailureGroup {
+func failureGroupList(report sippyprocessingv1.TestReport) []sippyv1.FailureGroup {
```

Comment:
@deads2k splitting these in separate commit will reduce the noise... next time :)